ui/src/components/Issuer/IssuedToken.tsx

```diff
@@ -197,9 +197,10 @@ const AllocationsChart = (props: { nettedTokenDeposits: DepositInfo[] }) => {
     )
 
     function formatNetTokenDeposits(tokens: DepositInfo[]): IDonutChartData[] {
+
         return tokens.map(t => {
             return {
-                title: `${t.investor}@${t.provider}`,
+                title: `${t.investor.replace('|','')}@${t.provider}`,
```

Comment:
Is it possible to replace the party IDs with party names for these chart labels? Would also help a lot with the readability and preventing overlaps